<launch>
  <param name="use_sim_time" value="true"/>
  <node pkg="loam_interface" type="ht_loamInterface" name="loamInterface" output="screen" required="true">
    <param name="stateEstimationTopic" type="string" value="/wamv/robot_localization/odometry/filtered" />
    <param name="registeredScanTopic" type="string" value="/velodyne_points" />
    <param name="flipStateEstimation" type="bool" value="false" />
    <param name="flipRegisteredScan" type="bool" value="false" />
    <param name="sendTF" type="bool" value="false" />
    <param name="reverseTF" type="bool" value="false" />
  </node>

</launch>
